client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/TemperatureChart.tsx

```diff
@@ -195,3 +197,19 @@ export const TemperatureChart = () => {
     </Box>
   );
 };
+
+export const TemperatureChart = () => {
```

Comment:
new wrapper component, so that I can easily show the toolbar, with filter, regardless of whether data is available or not. If the filters are set, so that no data is returned, we still want the filter to show.